src/test/java/org/jabref/logic/pdf/PdfAnnotationImporterTest.java

```diff
@@ -0,0 +1,130 @@
+package org.jabref.logic.pdf;
+
+import java.nio.file.Paths;
+import java.util.List;
+
+import org.jabref.model.pdf.FileAnnotation;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class PdfAnnotationImporterTest {
+
+    private AnnotationImporter importer = new PdfAnnotationImporter();
+
+    @Test
+    public void noAnnotationsWriteProtected() {
+
+        List<FileAnnotation> annotations = importer.importAnnotations(Paths.get("src/test/resources/pdfs/write-protected.pdf"));
+        assertEquals(0, annotations.size());
```

Comment:
it is preferable to check versus an empty list: `assertEquals(Collections.emptyList(), annotations)`. In the case, when the test fails you get a more helpful error message.